reference/configuration/framework.rst

```diff
@@ -1145,6 +1146,14 @@ The type of the resource to hint the loaders about the format. This isn't
 needed when you use the default routers with the expected file extensions
 (``.xml``, ``.yaml``, ``.php``).
 
+default_uri
+.........
+
+**type**: ``string``
+
+The default URI used to generate URLs in a non-HTTP context  (see
+:doc:`/routing`#generating-urls-in-commands).
```

Comment:
What is the correct syntax to link to: https://symfony.com/doc/current/routing.html#generating-urls-in-commands

Reply:
I am currently on a phone 📱 

But try :doc:'name </path/to/file>'

And use real backticks which I can't got to work on my phone 😜

Reply:
Ah you want to link to an anchor, in this case:

:ref: instead of :doc:'name <name-of-the-anchor>' and real backticks.

If no anchor is set before the headline in routing.rst add one:


.. _name-of-the-anchor:

My Headline
-----------